Makefile

```diff
@@ -0,0 +1,14 @@
+PYTHON ?= /usr/bin/python
```

Comment:
Use ```python``` instead of specifying a full path. That will make it work with alternative paths, such as when using a python virtual environment.